src/main/java/Duke.java

```diff
@@ -1,35 +1,56 @@
-/*
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
-
- */
-
 import java.util.Scanner;
 public class Duke {
     public static void printLine() {
         System.out.println("____________________________________________________________");
     }
     public static void main(String[] args) {
-        String line = " ";
+        String[] list = new String[100];
+        for (int i = 0; i < 100; i++) {
+            list[i] = " ";
+        }
+        int index = 0;
         Scanner in = new Scanner(System.in);
         printLine();
         System.out.println("Hello! I'm Doraemon!");
         System.out.println("What can I do for you?");
         printLine();
-        while (!line.equals("bye")) {
-            line = in.nextLine();
-            System.out.println(line);
+        while (!list[index].equals("bye")) {
+            list[index] = in.nextLine();
+            if (list[index].equals("bye")) {
+                break;
+            }
+            if (list[index].equals("list")) {
+                int counter = 1;
+                for (int j = 0; j < index; j++) {
+                    System.out.print(counter + ". ");
+                    System.out.println(list[j]);
+                    counter++;
+                }
+                index--;
+            } else {
+                System.out.print("Added: ");
+                System.out.println(list[index]);
+            }
             printLine();
+            index++;
         }
```

Comment:
Would it be better to set this as a separate function such as handleUserInput()?
```suggestion
handleUserInput()
```